.github/workflows/test-integration-windows.yml

```diff
@@ -0,0 +1,1191 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+name: test-integrations-windows
```

Comment:
can we rename the file to `test-integrations-windows` to match this name as well as the existing `test-integrations.yml` that it is based off?